qemu/tests/hugepage_specify_node.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+from virttest.staging import utils_memory
```

Comment:
please instead utils_memory with  avocado.utils.memory.

Reply:
Thinks a lot, correcting.

Reply:
@xutian  about "from virttest.staging import utils_memory"
I see in test_setup.py, its call also comes from "from virttest.staging import utils_memory"
So the "staging" file is called for the framework process level, so it is best not to use this at the case level. Is this so?

Reply:
It's better to used avocado API first  if  avocado-vt  has duplicate API. because  we have see some common module has begin to move from avocado-vt to  avocado framework.

Reply:
Thanks for the reminder.Corrected.